.github/workflows/ci.yml

```diff
@@ -34,6 +34,25 @@ jobs:
         run: |
           ./src/valkey-unit-tests
 
+  # Similar to the main build, but include additional feature coverage
+  test-ubuntu-latest-extra:
```

Comment:
I applied what I think makes sense for CI. @zuiderkwast if you have a second, please take a look.

Reply:
Yes, thanks. I see you modified 32-bit and macos jobs to enable fast float and not add new jobs for these. That's good. Adding 1 new job is better than adding 3, although I suggested adding zero.

I was suggesting enabling it `test-ubuntu-latest` too. We have for example the `test-sanitizer-address` and `test-ubuntu-latest-cmake ` that run without fast float. But it's fine to add this one job. We can maybe clean the jobs once a year or so.

Reply:
Yeah, I ended up with just one new test in the CI for now. I suppose you're saying the CI could do fast float and by default, and daily could hit both since we have like 8 different tests. I guess that makes sense, let me adapt that.